spec/lib/locasms/client_spec.rb

```diff
@@ -4,6 +4,40 @@
   let(:rest_client) { 'rest_client mock' }
   subject { LocaSMS::Client.new :login, :password, rest_client: rest_client }
 
+  context '.ENDPOINT' do
+    def test_when(moment_in_time)
+      Timecop.freeze(Time.parse(moment_in_time)) do
+        LocaSMS.send :remove_const, :Client
+        load 'lib/locasms/client.rb'
+        yield
+      end
+    end
+
+    it 'prior to Jun 1st 2015' do
+      test_when('2015-05-31T23:59:59-0300') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.